packages/terra-clinical-onset-picker/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "terra-clinical-onset-picker",
+  "version": "0.1.0",
+  "description": "A complete form for selecting an onset date. Dates can be selected with a precision of day, month, or year.",
+  "main": "lib/OnsetPicker.js",
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "rm -rf lib",
+    "compile:build": "$(cd ..; npm bin)/babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "$(cd ..; npm bin)/eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "release:major": "npm test && node ../../scripts/release/release.js major",
```

Comment:
These release scripts should be removed.

Reply:
Removed release scripts.